DEVELOPER_GUIDE.md

```diff
@@ -140,52 +140,54 @@ Following configurations are for WSO2 Api Manager 3.0.0 or newer versions. For o
-     |synapse.version|2.1.7-wso2v131|2.1.7-wso2v80|2.1.7-wso2v65|2.1.7-wso2v48|2.1.7-wso2v10|
-    
-2. Add the JAR file of the extension to the **<APIM_HOME>/repository/components/dropins** directory. 
-   You can find the org.wso2.carbon.apimgt.securityenforcer-\<version>.jar file in the **apim-handler-pingai/target** directory. 
+    org.wso2.carbon.apimgt.securityenforcer-\<version>.jar file can be found in **apim-handler-pingai/target** directory.
+
+    Use the following table to update pom.xml with the corresponding dependency versions for API manager.
+
+     | Dependency                |   APIM 3.0.0   |  APIM 2.6.0   |  APIM 2.5.0   |  APIM 2.2.0   |  APIM 2.1.0   |
+     | ------------------------- | :------------: | :-----------: | :-----------: | :-----------: | :-----------: |
+     | carbon.apimgt.version     |    6.5.349     |    6.4.50     |    6.3.95     |    6.2.201    |    6.1.66     |
+     | carbon.kernel.version     |     4.5.1      |    4.4.35     |    4.4.32     |    4.4.26     |    4.4.11     |
+     | carbon.governance.version |     4.8.10     |    4.7.29     |    4.7.27     |    4.7.23     |     4.7.0     |
+     | synapse.version           | 2.1.7-wso2v131 | 2.1.7-wso2v80 | 2.1.7-wso2v65 | 2.1.7-wso2v48 | 2.1.7-wso2v10 |
+
+2. Add the JAR file of the extension to the **<APIM_HOME>/repository/components/dropins** directory.
+   You can find the org.wso2.carbon.apimgt.securityenforcer-\<version>.jar file in the **apim-handler-pingai/target** directory.
 
 3. Add the bare minimum configurations to the *deployment.toml* file, which can be found in the
 **<APIM_HOME>/repository/conf** directory.
 
    ```
     [apim.ai_security]
-    operation_mode = "async"
+    operation_mode = "sync"
     sideband_request_endpoint = "ASE_SIDEBAND_REQUEST_ENDPOINT"
+    backup_sideband_request_endpoint = "BACKUP_ASE_SIDEBAND_REQUEST_ENDPOINT"
     ase_token = "SIDEBAND_AUTHENTICATION_TOKEN"
     model_creation_endpoint = "ASE_REST_API_ENDPOINT"
     access_key = "ASE_REST_API_ACCESS_KEY"
     secret_key = "ASE_REST_API_SECRET_KEY"
    ```
      **Note:**
-        
+
     - Select the Operation mode from **[sync](https://github.com/wso2-extensions/apim-handler-pingai/blob/master/DEVELOPER_GUIDE.md#sync-mode)**,
-        **[async](https://github.com/wso2-extensions/apim-handler-pingai/blob/master/DEVELOPER_GUIDE.md#async-mode)** and 
+        **[async](https://github.com/wso2-extensions/apim-handler-pingai/blob/master/DEVELOPER_GUIDE.md#async-mode)** and
         **[hybrid](https://github.com/wso2-extensions/apim-handler-pingai/blob/master/DEVELOPER_GUIDE.md#hybrid-mode)**.
-        If the mode is not set, the default mode is set as **async**. 
+        If the mode is not set, the default mode is set as **async**.
    - ASE_SIDEBAND_REQUEST_ENDPOINT : https://\<ase-host-machine-ip>:\<data-port>
+   - BACKUP_ASE_SIDEBAND_REQUEST_ENDPOINT : https://\<backup-ase-host-machine-ip>:\<data-port>
```

Comment:
ASE management api configuration will be synced between primary and back up hosts. So if we add API's to any one of the ASE hosts, it will be synchronised across.